dev-games/openscenegraph-openmw/openscenegraph-openmw-3.6_p20210610.ebuild

```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1,2} )
+
+WX_GTK_VER="3.0-gtk3"
+inherit cmake flag-o-matic lua-single wxwidgets
+
+DESCRIPTION="OpenMW-specific fork of OpenSceneGraph"
+HOMEPAGE="https://github.com/OpenMW/osg"
+MY_COMMIT="b02abe200c4847e73b887b064a89ea1758a5b733"
+SRC_URI="https://github.com/OpenMW/osg/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/osg-${MY_COMMIT}"
+
+LICENSE="wxWinLL-3 LGPL-2.1"
+SLOT="0/162" # NOTE: CHECK WHEN BUMPING! Subslot is SOVERSION
+KEYWORDS="~amd64 ~x86"
+IUSE="collada curl dicom debug doc egl examples ffmpeg fltk fox gdal
+gif glut gstreamer jpeg las lua openexr openinventor osgapps pdf png
+sdl sdl2 svg tiff truetype vnc wxwidgets xrandr +zlib"
```

Comment:
Kind of unusual so see this like this, you can indent like other variables with tabs.